src/main/java/com/minecolonies/coremod/event/ClientEventHandler.java

```diff
@@ -82,8 +83,9 @@ public void renderWorldLastEvent(@NotNull final RenderWorldLastEvent event)
                 final Entity entityCitizen = world.getEntityByID(citizen.getEntityId());
                 if (entityCitizen instanceof EntityCitizen)
                 {
-                    RenderUtils.renderSigns(world, event.getPartialTicks(), citizen, player,
-                      entityCitizen.getPosition());
+                    //TODO: Reimplement status rendering.
+                    //RenderUtils.renderSigns(world, event.getPartialTicks(), citizen, player,
+                    //  entityCitizen.getPosition());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)